vllm/config.py

```diff
@@ -184,28 +185,32 @@ def _verify_tokenizer_mode(self) -> None:
     def _verify_sparsity(self) -> None:
         supported_sparsity = ["sparse_w16a16", "semi_structured_sparse_w16a16"]
 
+        hf_sparsity_config = getattr(self.hf_config, SPARSITY_CONFIG_NAME,
```

Comment:
@robertgshaw2-neuralmagic just making sure that this is the proper flow.
The compressed model's HF config will look like this:

```bash
"sparsity_config": {
    "format": "sparse_bitmask",
    "global_sparsity": 38.77294698534069,
    "sparsity_structure": "0:0"
  },
```

The logic in this method will look for the "sparsity_structure" key (as defined in `compressed-tensors`) and check it against the `supported sparsity`.

Reply:
We should make sure that there is a clear mapping from our 'sparsity_structure' field (`sparseml` ) to `supported_sparsity` here in nm-vllm. @bfineran , @Satrat , @robertgshaw2-neuralmagic potential blind spot for us to cover asap.